configuration/defaults.conf

```diff
@@ -0,0 +1,6 @@
+BACKEND_BUCKET_NAME=xplorers-artifacts
+BACKEND_BUCKET_STORAGE_CLASS=STANDARD
+BACKEND_BUCKET_TERRAFORM_PREFIX=xplorers/infra/terraform/state
+GOOGLE_CLOUD_PROJECT_ID=pras-sandbox
+GOOGLE_CLOUD_PROJECT_REGION=us-central1
+GOOGLE_CLOUD_PROJECT_ZONE=us-central1-c
```

Comment:
I am not too sure. But can these variables comes from `.env` file. Please correct me if I'm wrong

Reply:
They are imported in [Makefile](https://github.com/xplorer-io/xplorers-gcloud-infra-bootstrap/pull/1/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R5) and are available for any `make` commands. This way, we only define these in one place and they get used everywhere else.

Reply:
These are configuration variables not secrets hence should be ok. This way variables are version controlled and audited.

https://99devops.com/configuration-variables-and-secret-variables-are-two-different-things-stop-treating-them-as-one